### PR TITLE
Commit most recent version of worker metadata to repo

### PR DESCRIPTION
**Pull request by cicdw**
For 2.14.13

### CODE REVIEW

File: src/prefect/server/api/collections_data/views/aggregate-worker-metadata.json [hunk 1]
```diff
@@ -180,9 +180,14 @@
             },
             "launch_type": {
               "title": "Launch Type",
-              "description": "The type of ECS task run infrastructure that should be used. Note that 'FARGATE_SPOT' is not a formal ECS launch type, but we will configure the proper capacity provider strategy if set here.",
+              "description": "The type of ECS task run infrastructure that should be used. Note that 'FARGATE_SPOT' is not a formal ECS launch type, but we will configure the proper capacity provider stategy if set here.",
```
Comment (zzstoatzz) [inline src/prefect/server/api/collections_data/views/aggregate-worker-metadata.json:183]:
```suggestion
              "description": "The type of ECS task run infrastructure that should be used. Note that 'FARGATE_SPOT' is not a formal ECS launch type, but we will configure the proper capacity provider strategy if set here.",
```

Comment (cicdw) [reply src/prefect/server/api/collections_data/views/aggregate-worker-metadata.json:183]: I actually can't change that here - the release workflow dynamically injects this file as-is into our build, which causes our versioning to break, so I'm committing the most current version of the file so the release succeeds. We most likely need to setup a GitHub action that PRs to `main` anytime the file in the other repo is updated

Comment (zzstoatzz) [reply src/prefect/server/api/collections_data/views/aggregate-worker-metadata.json:183]: ohhhh ok nevermind, ill just fix it upstream and the correct version will come in next time

**Comment by cicdw** [comment]
We are going to have to ignore the static analysis failure for this (see comment above)

**Review by zzstoatzz — APPROVED** [review]

**Pull request merged**